public/assets/js/index.js

```diff
@@ -0,0 +1,5 @@
+var now = new Date()
+var year = now.getFullYear()
+
+document.querySelector('.footer-copyright span').
```

Comment:
Let's change this a little bit, so we only change the year, and leave the rest of the string untouched.
SO we are going to create a new element on the footer-copyright, with class `copyright-year` for example

```suggestion
document.querySelector('.footer-copyright .copyright-year').innerHTML = year
```

Reply:
Also, when you concatenate things, the dot (`.`) should be always on a new line
For example:
```js
document.querySelector('.element')
  .stuff1
  .stuff2
```

Reply:
🤔 got it.